companion-snap-compose/start.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eu
+
+echo "Running: docker compose -f $SNAP/docker-compose.yaml up"
+docker compose -f $SNAP/docker-compose.yaml up
+# docker compose up
```

Comment:
Code in comments is redundant and hard to maintain. Turn on xtrace instead?

```suggestion
#!/bin/bash -eux

docker compose -f $SNAP/docker-compose.yaml up
```

Reply:
I'm just removing the echo, as it does not really provide any benefits.